ichnaea/data/tests/test_station.py

```diff
@@ -89,8 +93,15 @@ def test_lock_timeout(
             lac=obs.lac,
             cid=obs.cid,
             samples=10,
+            created=datetime(2019, 12, 1, tzinfo=UTC),
         )
         session2.add(cell)
+        # flush sends the INSERT to the database, but does not commit it.
+        # This cell is seen as an existing cell the first pass through
+        # the task. However, because of the rollback inside the task, it is
+        # removed from the database, and the observation inserts it as a new
+        # cell. This is probably an error in the test, but it retained because
+        # it is a working historical test.
```

Comment:
I'd support removing this test entirely. It covers the same issue as ``test_retriable_exceptions``, and generates the error in a more "natural" way, but it also relies on the buggy behaviour around session rollbacks.

Reply:
If you think the thing we need to test is covered by another test, I'm totally game for removing it.